specification/indices/refresh/IndicesRefreshRequest.ts

```diff
@@ -28,6 +28,8 @@ import { ExpandWildcards, Indices } from '@_types/common'
  * By default, Elasticsearch periodically refreshes indices every second, but only on indices that have received one search request or more in the last 30 seconds.
  * You can change this default interval with the `index.refresh_interval` setting.
  *
+ * In Elastic Cloud Serverless, the default refresh interval is 5 seconds.
```

Comment:
Is it always 5 seconds, or do you also need to receive at least one search request in the last 30 seconds?